requirements.txt

```diff
@@ -12,17 +12,35 @@ apscheduler==3.9.1 \
     --hash=sha256:65e6574b6395498d371d045f2a8a7e4f7d50c6ad21ef7313d15b1c7cf20df1e3 \
     --hash=sha256:ddc25a0ddd899de44d7f451f4375fb971887e65af51e41e5dcf681f59b8b2c9a
     # via flask-apscheduler
-babel==2.10.1 \
-    --hash=sha256:3f349e85ad3154559ac4930c3918247d319f21910d5ce4b25d439ed8693b98d2 \
-    --hash=sha256:98aeaca086133efb3e1e2aad0396987490c8425929ddbcfe0550184fdc54cd13
+babel==2.10.3 \
```

Comment:
Why are those requirements changed as well? Shouldn't be affect by hwi and embit.

Reply:
Babel is currently contrained by: `">6.0.0,<7.0.0"`. If you delete the `requirements.txt`, you'll get the newest version. Otherwise, the version stays the same.